src/invoke.ts

```diff
@@ -0,0 +1,15 @@
+import { promises } from '@api3/operations-utilities/dist/index';
```

Comment:
I've fixed operations-utilities (and published it) so you no longer need the full path. Feel free to give it a go.
```suggestion
import { promises } from '@api3/operations-utilities';
```

Reply:
I've also published operations-utilities as  "@api3/operations-utilities": "0.0.1-e4bcdbd569e8802192b41f2bb88ed0cca7fc0c7b-1"

We'll probably have to change the name soon due to a possible conflict/confusion relating to the split of the operations repo. Feel free to suggest an alternative 🤷